### PR TITLE
my solution for #1

### PR DESCRIPTION
**Pull request by johnklee**
Selina's homework1

### CODE REVIEW

**Review by johnklee — COMMENTED** [review]
* 可以通做測試 (Pass)
* 某些代碼需要被移除. (e.g.: 移除 `# TBD`)
* 某些代碼可以改善 (e.g.: `if customer_type == CustomerType(3):`)
* 類別 **CounterV2** 設計可以更好

File: homework01/counter.py [hunk 1]
```diff
@@ -23,7 +24,16 @@ def count(self, prices:List[int]):
...
+    end_prices = 0
+    if prices[0] < 100:
+      end_prices = prices[0]
+    elif prices[0] >= 100 and prices[0] <= 200:
+      end_prices = prices[0] * 0.9
+    elif prices[0] > 200 and prices[0] <=500:
+      end_prices = prices[0] * 0.8
+    elif prices[0] > 500:
+      end_prices = max(400, prices[0] * 0.6)
+    return math.floor(end_prices)
```
[the remark this answers is not in the document]
Comment (selinahuang55) [reply homework01/counter.py:36]: Got it.

File: homework01/counter.py [hunk 2]
```diff
@@ -23,7 +24,16 @@ def count(self, prices:List[int]):
     If float number is obtained after calculation, use floor to translate it into integer.
     '''
     # TBD
```
[the remark this answers is not in the document]
Comment (selinahuang55) [reply homework01/counter.py:26]: Removed.

File: homework01/counter.py [hunk 3]
```diff
@@ -48,4 +58,13 @@ def count(self, prices:List[int], customer_type:CustomerType):
     - Otherwise, no discount at all.
     '''
     # TBD
```
[the remark this answers is not in the document]
Comment (selinahuang55) [reply homework01/counter.py:60]: Removed.

**Pull request merged**